src/main/resources/assets/chisel/lang/es_es.lang

```diff
@@ -0,0 +1,2871 @@
+item.chisel.chisel_iron.name=Cincel
+item.chisel.chisel_diamond.name=Cincel de Diamante
+item.chisel.chisel_hitech.name=iChisel
+item.chisel.offsettool.name=Varita de Diseños Ender
```

Comment:
What about "Varita de Desplazamiento de Ender"?
I'd say it fits better with the original "Ender Offset Wand"
(Also propagate this change to L56)

Reply:
Okay! then i´ll change it as soon as possible